staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -542,10 +557,23 @@ func kmsPrefixTransformer(config *apiserverconfig.KMSConfiguration, stopCh <-cha
 			l:            &sync.Mutex{},
 			lastResponse: &kmsPluginHealthzResponse{},
 		}
+		// initialize keyID
+		keyID := ""
+		probe.keyID.Store(&keyID)
+
+		go wait.PollImmediateUntilWithContext(
+			ctx,
+			time.Minute,
+			func(ctx context.Context) (bool, error) {
+				if err := probe.check(ctx); err != nil {
+					klog.V(2).Infof("PollImmediateUntilWithContext calling probe check() err: %v", err)
```

Comment:
```suggestion
					klog.V(2).InfoS("PollImmediateUntilWithContext calling probe check()", "err",  err)
```

Need structured logging